packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -1500,7 +1500,7 @@ abstract class BaseSliderTrackShape {
 /// [parentBox]. The track rectangle extends to the bounds of the [parentBox],
 /// but is padded by the [RoundSliderOverlayShape] radius. The height is defined
 /// by the [SliderThemeData.trackHeight]. The color is determined by the
-/// [Slider]'s enabled state and the track segments's active state which are
+/// [Slider]'s enabled state and the track segments' active state which are
```

Comment:
`segment's`?

Reply:
Aaah, when I was doing my spell checking, I thought that the comment was referring to a plural number of segments' active state, not just one segment's active state.

Clearly not so.  Changed to `segment's`